server/fleet/app.go

```diff
@@ -172,8 +172,11 @@ func (m MacOSUpdates) Validate() error {
 
 // MacOSSettings contains settings specific to macOS.
 type MacOSSettings struct {
-	CustomSettings       []string `json:"custom_settings"`
-	EnableDiskEncryption bool     `json:"enable_disk_encryption"`
+	// CustomSettings is a slice of configuration profiles to apply to enrolled devices.
```

Comment:
Just to be sure I'm not missing anything, these comments are not related to the PR and were done "on the fly". right?

Reply:
If so, that's a good thing 🙂 . Just want to be sure.

Reply:
Correct. 👍